tests/Baseline/spicy.rt.confirmation/output

```diff
@@ -0,0 +1,13 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+---
+ok
+---
+fail
+---
+ok
+fail
+---
+fail
+ok
```

Comment:
I am confused by the semantics of (the absence of) `--require-input` here. Shouldn't the driver stop feeding input after `spicy::reject_input` is called?

Reply:
You mean `--require-confirmation` (or now: `—require-accept`)? That's there, but the driver doesn't stop right now, it's only used for deterring success in the end. We could change that, but not immediately sure if that's better.